www/content/comparatif/index.md

```diff
@@ -194,6 +205,12 @@ elle est *utilisable*, mais pas *optimisée* pour ça.
 - pour les Bépoètes qui ont des douleurs ou qui souhaitent tenter l’expérience
   [1DFH][], [Bépolar][] sera une alternative intéressante et très vite apprise.
 
+[ergo‑l]:     /
+[bépo]:       /bepo
+[erglace]:    /erglace
+[lafayette]:  /lafayette/#qwerty-lafayette
+[colemak ft]: /lafayette/#colemak-french-touch
+[bépolar]:    /lafayette/#bepolar
```

Comment:
Those tags are duplicates and generates errors when building the site. I’d say we move all of the link tags at the bottom of the page, but I’ll let you be the judge.

Reply:
Indeed, this creates warnings. Let’s drop the dupes then.